src/cargo_toml.rs

```diff
@@ -106,10 +106,24 @@ pub struct RemainingCoreMetadata {
     pub name: Option<String>,
     /// The directory containing the wheel data
     pub data: Option<String>,
+    /// Cargo compile targets
+    pub targets: Option<Vec<CargoTarget>>,
     #[serde(flatten)]
     pub other: HashMap<String, toml_edit::easy::Value>,
 }
 
+/// Cargo compile target
+#[derive(Serialize, Deserialize, Debug, Clone, Default)]
+#[serde(rename_all = "kebab-case")]
+pub struct CargoTarget {
+    /// Name as given in the `Cargo.toml` or generated from the file name
+    pub name: String,
+    /// Kind of target ("bin", "lib")
+    pub kind: Option<String>,
+    /// Bridge model, which kind of bindings to use
+    pub bindings: Option<String>,
```

Comment:
Unused for now, prepare for https://github.com/PyO3/maturin/issues/368#issuecomment-1220187210